x-pack/plugins/index_lifecycle_management/public/application/services/policies/policy_validation.ts

```diff
@@ -162,6 +173,7 @@ export const validatePolicy = (
       hot: hotPhaseErrors,
       warm: warmPhaseErrors,
       cold: coldPhaseErrors,
+      frozen: frozenPhaseErrors,
```

Comment:
I think `Object.keys(frozenPhaseErrors).length === 0` should be added to `isValid` variable calculation, otherwise the form will be submitted even when there are validation errors in frozen phase.